src/ragQueries.ts

```diff
@@ -0,0 +1,42 @@
+import { GraphAnnotation } from "./agent/agent"
```

Comment:
lets move this file into the `agent/` folder, the stuff outside `agent/` is more related to bullmq and the job processing.